R/get_osw_query.R

```diff
@@ -254,8 +316,36 @@ getPrecursorsQuery <- function(runType = "DIA_Proteomics", level = "Peptide")
       WHERE PRECURSOR.DECOY = 0
       ORDER BY peptide_id, transition_group_id, transition_id;"
     }
-  } else if (runType == "DIA_Metabolomics")
-  {
+  } else if (runType == "DIA_IPF"){
+    ## TODO: Not as clear on PROTEIN level, for now we only do peptide level
+      query <- "SELECT DISTINCT PRECURSOR.ID AS transition_group_id,
+      TRANSITION_PRECURSOR_MAPPING.TRANSITION_ID AS transition_id,
+      PEPTIDE.ID AS peptide_id,
+      PEPTIDE.MODIFIED_SEQUENCE AS sequence,
+      PRECURSOR.CHARGE AS charge,
+      PRECURSOR.GROUP_LABEL AS group_label
+      --TRANSITION.DETECTING AS detecting,
```

Comment:
What does `--` do?

Reply:
The `--` in sql is to signify a comment, like `#` in R or `//` in C++